config/optional/config_ignore.settings.yml

```
@@ -0,0 +1,8 @@
+mode: simple
+ignored_config_entities:
+  - 'block.block.*'
+  - exclude_node_title.settings
+  - 'system.menu.*'
+  - system.site
+  - 'taxonomy.vocabulary.*'
+  - 'webform.webform.*'
```

A more complete list of configs that content admins/editors have access to would look like this.  We need to discuss whether we should ignore all of these or not.
```suggestion
  - 'block.block.*'
  - block_class.settings
  - easy_breadcrumb.settings
  - exclude_node_title.settings
  - 'language.content_settings.*.*'
  - 'language.entity.*'
  - 'metatag.metatag_defaults.*'
  - 'shortcut.set.*'
  - 'system.menu.*'
  - system.site
  - 'taxonomy.vocabulary.*'
  - 'webform.webform.*'
  - xmlsitemap.settings
  - 'xmlsitemap.settings.*.*'
  - 'xmlsitemap.xmlsitemap.*'
```